core/src/main/java/de/danoeh/antennapod/core/feed/FeedMedia.java

```diff
@@ -199,6 +208,18 @@ public void setSize(long size) {
         this.size = size;
     }
 
+    /**
+     * Indicates we asked the service what the size was, but didn't
+     * get a valid answer and we shoudln't check using the network again.
+     */
+    public void setCheckedOnSize() {
```

Comment:
I do not have a better solution yet, but this makes it a bit hard to see that this information is persisted in the database.
I'd rather have something like ``setSize(CHECKED_BUT_UNKNOWN)`` [Basically, rename ``CHECKED_ON_SIZE`` to something that indicates that the fiel size was checked and that it could not be retrieved]

But would not really be a problem (apart from code readability, imho), if you leave it this way.